### PR TITLE
skip adding logforwarder on hosts if they have membership with cluste…

### PR DESCRIPTION
**Pull request by arnej27959**
…r type admin

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@bratseth please review
@jonmv FYI

### CODE REVIEW

File: config-model/src/main/resources/schema/admin.rnc [hunk 1]
```diff
@@ -105,6 +105,7 @@ ClusterControllers = element cluster-controllers {
 }
 
 LogForwarding = element logforwarding {
+    attribute unconditional { xsd:boolean }? &
```
Comment (bratseth) [inline config-model/src/main/resources/schema/admin.rnc:108]: "all"? "include-admin"?

**Review by bratseth — COMMENTED** [review]
Not sure if "unconditional" is a good name, will we expose this?
Otherwise LGTM

**Comment by arnej27959** [comment]
@bratseth please take another look

File: config-model/src/main/resources/schema/admin.rnc (hunk 1 above)
Comment (arnej27959) [reply config-model/src/main/resources/schema/admin.rnc]: include-admin sounds good.  it was mostly added in case it was needed for some unit test :-)

**Review by bratseth — APPROVED** [review]

**Pull request merged**